build.assets/genproto.sh

```diff
@@ -14,7 +14,8 @@ main() {
   rm -fr api/gen/proto gen/proto  # cleanup gen/proto folders
 
   # Generate Gogo protos (default).
```

Comment:
```suggestion
  # Generate Gogo protos.
  # Prefer using protoc-gen-go (see block below).
```

We are specializing paths, so there's no default anymore.